installer/windows-installer.iss

```diff
@@ -134,6 +135,8 @@ Root: HKLM; Subkey: "Software\Classes\.osp"; ValueType: string; ValueName: ""; V
 Root: HKLM; Subkey: "Software\Classes\OpenShotProject"; ValueType: string; ValueName: ""; ValueData: "{#MyAppProjectFileDesc}"; Flags: uninsdeletekey; Tasks: fileassoc
 ; Launcher association for data files of type OpenShotProject
 Root: HKLM; Subkey: "Software\Classes\OpenShotProject\shell\open\command"; ValueType: string;  ValueName: ""; ValueData: """{app}\{#MyAppExeName}"" ""%1"""; Tasks: fileassoc
+; Clean-up old, incorrect AppID Uninstaller (used in 2.6.1-dev builds)
```

Comment:
For a while now our daily builds have been using a different AppID key in the registry, missing { } curly brackets. This line removes this old entry from the registry.